R/read-data.R

```diff
@@ -178,6 +178,11 @@ read_anc_testing <- function(file) {
     val[["anc_known_neg"]] <- 0
   }
 
+  if ( !("births_facility" %in% names(val)) ||
+       all(is.na(val[["births_facility"]])) ) {
```

Comment:
This line is redundant since setting default value to `NA` now: `all(is.na(val[["births_facility"]]))`.  

Maybe remove this part of the condition for clarity?